WordPress/Classes/ViewRelated/Me/App Settings/Boolean User Defaults/BooleanUserDefaultsDebugViewModel.swift

```diff
@@ -16,40 +16,51 @@ final class BooleanUserDefaultsDebugViewModel: ObservableObject {
         }
     }
 
-    @Published var userDefaultsSections: Sections = []
+    @MainActor @Published private(set) var userDefaultsSections: Sections = []
 
-    init(coreDataStack: CoreDataStack = ContextManager.shared,
+    init(coreDataStack: CoreDataStackSwift = ContextManager.shared,
```

Comment:
The `CoreDataStackSwift` is the same as `CoreDataStack` protocol, but it provides `async` APIs.